server/routes/manageConditions/handlers/additionalLicenceConditionInput.ts

```diff
@@ -83,4 +83,19 @@ export default class AdditionalLicenceConditionInputRoutes {
 
     return res.redirect(redirect)
   }
+
+  SKIP = async (req: Request, res: Response): Promise<void> => {
+    const { user, licence } = res.locals
+    const { conditionId } = req.params
+
+    const condition = licence.additionalLicenceConditions.find(c => c.id === parseInt(conditionId, 10))
+    const conditionData = { conditionSkipped: '[DATE REQUIRED]' }
```

Comment:
meh... could call this `DATE_REQUIRED` and then it would be easier to migrate to an enum?

Reply:
I'm not sure what you mean by being easier to migrate to an enum, but I left it as `conditionSkipped` to make it slightly easier should we wish to add skipping functionality to any other conditions in future (that aren't necessarily date-specific) while leaving it generic in the DB.

Reply:
yes, totally agree with the variable named `conditionSkipped` - just meant if the value of the variable was `DATE_REQUIRED` then if we wanted to map it to an enum in the future then it would be trivial to do so. 
 
on the kotlin side: `val conditionSkipped = ConditionSkipReason.DATE_REQUIRED` 

(UPPER_CAMEL_CASE is a common way to represent codes/constants across programming languages, `[UPPER CASE]` is a bit bespoke)